src/fides/api/ops/models/policy.py

```diff
@@ -469,6 +487,95 @@ def update(self, db: Session, *, data: Dict[str, Any]) -> FidesBase:
         return super().update(db=db, data=data)
 
 
+class RuleUse(Base):
```

Comment:
Copy-pasted and adapted from RuleTarget, this is just for consent.  RuleUse may not be the best name - I'm not well-versed in the nomenclature here.

Reply:
Did you consider updating the `RuleTarget` table to include `.data_use` as an option vs. adding another table that's conditionally used alongside `ConsentRule`s? Might be cleaner if all "targets" are grouped together in some way.

Reply:
I did look into this but my first impression was that it was a little messier and more conditional logic would be needed than was desirable!  

I'd have to change `RuleTarget.data_category` to make it nullable but then add code-level logic everywhere to make sure we had data categories for access/erasure rules but not consent rules.

Definitely worth another look though given your opinion here, so I can explore how much additional validation is needed.

Reply:
Latest commits remove the `RuleUse` table entirely and just load the `config.json` file directly to see whether the customer has said a given `consentOption` is executable.